Makefile

```diff
@@ -89,7 +89,8 @@ plugin/types/%.pb.go: proto/%.proto
 
 ui: dkron/ui-dist
 
-main: dkron/ui-dist plugin/types/dkron.pb.go plugin/types/executor.pb.go *.go */*.go */*/*.go */*/*/*.go
+#main: dkron/ui-dist plugin/types/dkron.pb.go plugin/types/executor.pb.go *.go */*.go */*/*.go */*/*/*.go
```

Comment:
Remove comment?

Reply:
It's for local test. 
And sorry that I missed this message, a new pull request, please take a review.